packages/venia-concept/src/actions/cart/asyncActions.js

```diff
@@ -241,9 +241,16 @@ export const getCartDetails = (payload = {}) => {
 
             // for each item in the cart, look up its image in the cache
             // and merge it into the item object
+            // then assign its options from the totals subResource
             if (imageCache && Array.isArray(items) && items.length) {
                 items.forEach(item => {
                     item.image = item.image || imageCache[item.sku] || {};
+
+                    item.options =
+                        totals && totals.items
```

Comment:
Since this conditional doesn't change with each `item`, you could pull this out of the `forEach` if you wanted. Doesn't block a 👍 though.

Reply:
Oh of course! Ok, I'll make that quick change and get this pushed. Thanks again!

Reply:
@supernova-at should be good to go now!